algorithms/scripts/SGDClassifier-master.py

```diff
@@ -35,7 +35,7 @@
 # In[3]:
 
 # We're going to be building a 'TP53' classifier 
-GENE = 'TP53'
+GENE = '7157' # TP53
```

Comment:
I had forgotten to delete `scripts/1.TCGA-MLexample.py`. I did a force push. Now see the diff for "scripts/1.TCGA-MLexample.py → scripts/2.TCGA-MLexample.py"